benchmark/README.md

```diff
@@ -60,9 +60,6 @@ TPCH timing results is written to stdout in the following form: `TPCH_Result,<la
     <true for sql tests, false for functional tests>
     ```
 
-    **Note**: Ensure that you build the worker and application with .NET Core 3.0 in order to run hardware acceleration queries.

```

Comment:
This note should still be here with the `.NET 6` instead of `.NET Core 3.0` since `net461` is still part of the targetframeworks for the worker and benchmark app.

Reply:
Restored